nostrpostrlib/src/main/java/nostr/postr/Relay.kt

```diff
@@ -64,38 +66,51 @@ class Relay(
                 // As a rough guess, we assume we might have missed events during those last 30s ...
                 // TODO: Make sure that queries that have not gotten their EOSE yet, get resent unmodified.
                 val lastGood = System.currentTimeMillis() - 30_000
-                connect(lastGood)
+                connect(subscriptionId = subscriptionId, reconnectTs = lastGood)
```

Comment:
Note: I think `onClosing()` might be problematic. You'll see that in your app but as it is right now, you might reconnect even if the closing was triggered from the app itself.

Reply:
So, should I remove it?